### PR TITLE
Update ods-core.env.sample

### PR DESCRIPTION
**Pull request by felipecruz91**
Added Nexus, SonarQube and ProvApp resource request/limit link that points to the templates - (Issue #817)

[Jenkins](https://github.com/opendevstack/ods-core/blob/master/jenkins/ocp-config/build/bc.yml#L50-L56) and [DocGen](https://github.com/opendevstack/ods-core/blob/master/ods-document-generation-svc/ocp-config/bc.yml#L36-L42) resources are hard-coded unfortunately.

### CODE REVIEW

File: configuration-sample/ods-core.env.sample [hunk 1]
```diff
@@ -46,6 +46,10 @@ NEXUS_PASSWORD_B64=changeme
 # Nexus (basic) authentication username:password
 NEXUS_AUTH=developer:changeme
 
+# Nexus request/limits.
+# You can configure them here: https://github.com/opendevstack/ods-core/blob/master/nexus/ocp-config/bc.yml#L14-L21
+# For instance,
+# NEXUS_BUILD_CPU_REQUEST=200m
```
Comment (michaelsauter) [inline configuration-sample/ods-core.env.sample:52]: For me this sounds like people should edit https://github.com/opendevstack/ods-core/blob/master/nexus/ocp-config/bc.yml#L14-L21. But that's not really what we want ... I think we should tell the user that all params in that template can be overwritten by specifying the param in this file here ...

**Comment by michaelsauter** [comment]
@felipecruz91 Do you want to make the hardcoded instance parameterised as well? Would be good for consistency ..

File: configuration-sample/ods-core.env.sample (hunk 1 above)
Comment (felipecruz91) [reply configuration-sample/ods-core.env.sample]: That's right - it gives a false impression that they should edit the params, well spotted!

**Comment by felipecruz91** [comment]
> @felipecruz91 Do you want to make the hardcoded instance parameterised as well? Would be good for consistency ..

@michaelsauter Sure! I'll be creating 2 separate PRs, one for Jenkins and another one for Docgen.

**Review by michaelsauter — APPROVED** [review]

**Pull request merged**